src/gui/MainWindow.cpp

```diff
@@ -89,8 +89,9 @@ MainWindow* getMainWindow()
     return g_MainWindow;
 }
 
-MainWindow::MainWindow()
+MainWindow::MainWindow(bool startMinimized)
```

Comment:
I don't like messing with the constructor, this should really be done by a setter function. You can probably just call hideWindow() directly from main right after construction.

Reply:
I don't believe I can. At least I tried and it didn't work. There are multiple codepaths which do show the window, my main suspect is `MainWindow::restoreConfigState()`.

Another option is to create a QTimer in main.cpp with a short (or even zero) timeout and then call `hideWindow()` from within. But that feels equally hacky, tbh.

Reply:
Agreed, perhaps this is the only acceptable solution. I'll let it stand for now :-)